myapp/template.html

```diff
@@ -1,36 +1,77 @@
+            del salario mínimo en México. De acuerdo con estos autores, en la segunda mitad del siglo XX, se identifican
+            tres etapas en la evolución de los salarios mínimos en México, la primera inicia en 1950 y terminaría para
+            la segunda mitad de la década de 1970, en esta etapa se caracterizó por cuadruplicar el valor del salario
+            mínimo. No obstante, para 1977 los incrementos al salario mínimo perdieron ritmo y comenzaron a rezagarse
+            ligeramente respecto de la inflación. Sobre esta etapa se señala que en términos reales el salario mínimo
+            creció entre 1969 y 1977 a un ritmo de 5.7% anual y entre el periodo de 1977 a 1982 disminuyó a 1.3%.
+            Entre 1981y 1982 el país enfrentó problemas económicos severos y, el gobierno mexicano inició lo que
+            Camberos y Bracamontes (2015) llaman las políticas de ajuste. Con estas políticas se inician los procesos de
+            apertura económica y globalización para México y la segunda etapa del salario mínimo. Esta etapa corresponde
+            al periodo 1982 y 1992 y en el cual el salario mínimo decreció a un ritmo de 6.3% anual, para 1994 el
+            salario mínimo había perdido el 66% de su poder adquisitivo. La última etapa corresponde al periodo 1995 y
+            2014, en esta etapa el salario mínimo se mantuvo a la baja, aunque no en las mismas magnitudes que las
+            etapas precedentes. Se observa que entre 1995 y 2003 el salario mínimo decreció a un ritmo de -2.5% anual, y
+            finalmente, el periodo entre 2004 y 2014 corresponde a la tercera etapa del salario mínimo no creció ni
+            decreció, se mantuvo constante.
+            Es posible observar que el modelo económico en México se ha destacado por una política de contención
+            salarial que ha significado la pérdida dramática del poder adquisitivo y un deterioro sistemático de los
+            salarios mínimos, en México no son suficientes para cubrir las necesidades de las personas. Las iniciativas
+            para incrementar los salarios mínimos se ven truncadas con el argumento teórico de que un incremento del
+            salario solo puede tener lugar cuando existen aumentos en la productividad del trabajo de otro modo solo se
+            lograría destruir el empleo. Sin embargo, se observa que la productividad ha mostrado una tendencia positiva
+            no se ha reflejado en aumentos de los salarios. Hay quienes señalan que esta desvinculación entre el salario
+            mínimo y la productividad del trabajo se debe a el salario mínimo ha sido utilizado como un instrumento
+            antinflacionario y al mismo tiempo como una ventaja competitiva que ha servido la integración de la
+            económica mexicana con el mundo, (Davila, Perez y Livas 1992), (Camberos y Bracamontes 2015), (Moreno Brid
+            et al. 2014).
+        </p>
+    </div>
+    <div>
```

Comment:
Aquí hay dos cambios. El primero es para que no se vaya a ocupar toda la pantalla y  el segundo para no agregar alguna característica especial a la figura (pero la neta no sé cuál es esa característica):
```suggestion
    <div class="container">
        <figure class="figure">
```